optimizely/helpers/condition.py

```diff
@@ -486,7 +508,7 @@ def semver_greater_than_evaluator(self, index):
 
         return result > 0
 
-    def semver_less_than_evaluator(self, index):
+    def semver_less_than_evaluator(self, index: int) -> Optional[bool]:
```

Comment:
Why it's optional bool?

Reply:
Doc string says:
https://github.com/optimizely/python-sdk/blob/9b4ec4f109cf4eac33b64e7fc425278e73f7f420/optimizely/helpers/condition.py#L517-L522